### ISSUE TITLE
Code Review - Ben

### ISSUE THREAD

**Issue by WoodsyTheNerd** [a-terlik/OOP-CODE-REVIEW-ASSESSMENT#1]
### 👍Done Well
- Efficient implementation of each shuffling type. 
- Code works clearly with well defined and intended inputs
- Code is easy to read and understand

### ⚒️ Suggestions
- Some form of error handling for erroneous inputs would be beneficial 
![Screenshot 2023-03-08 161227](https://user-images.githubusercontent.com/119528792/223767287-b456be85-39d2-4e06-b7b7-59fa3223d151.png)
- Some kind of class or function to deal X many cards instead of the entire deck might be helpful (as was guided by the brief). This could be build with something a function like this:
```
public static List<T> Deal<T>(string CardsToBeDealt, List<T> Deck)
        {
            // Final Iteration 
            int length = Int32.Parse(CardsToBeDealt);
            for (int i = 0; i < length; i++)
            {
                Console.WriteLine(Deck[i]);
            }

            return Deck;

        }    
```
```
string CardsToBeDealt = Console.ReadLine();
            Deck = Deal(CardsToBeDealt, Deck);
```

Tl;dr: highly efficient code for intended purpose. Would benefit from stronger error handling
